index.html

```diff
@@ -5049,6 +5049,25 @@ <h3>Issuer Cooperation Impacts on Privacy</h3>
 <a>presentations</a> of the <a>credential</a>, regardless of a <a>verifier</a>'s
 inability to do so.
         </p>
+        <p>
+In addition to previously described privacy protections an <a>issuer</a> may use,
+<a>issuers</a> should also be aware of data they leak associated with identifiers
+and claim types they use when issuing <a>credentials</a>. One example of this would
+be an <a>issuer</a> issuing drivers licenses which reveal both the location(s) in
+which they have jurisdiction and the location of the <a>subject's</a> residence.
+<a>Verifiers</a> may take advantage of this by requesting a <a>credential</a>
+ostensibly to check that the <a>subject</a> is licensed to drive a limousine, when
+in fact they are interested in metadata <em>about</em> the credential, such as which
+<a>issuer</a> issued the credential, and tangential information that may have been
+leaked by the <a>issuer</a>, such as the subject's home address. To mitigate such
+leakage, <a>issuers</a> might choose to use common identifiers to mask specific 
+location information or other sensitive metadata, e.g., a shared issuer at a state
+or nation level, instead of at the level of a county, city, town, or other smaller
+municipality. Further, <a>holder</a> attestation mechanisms can be used by
+<a>verifiers</a> to preserve privacy, by providing proofs that an <a>issuer</a>
+exists in a set of trusted entities, without needing to disclose the exact
+<a>issuer</a>.
```

Comment:
```suggestion
<a>Verifiers</a> might take advantage of this by requesting a <a>credential</a>
to check that the <a>subject</a> is licensed to drive, when
in fact they are interested in metadata <em>about</em> the credential, such as which
<a>issuer</a> issued the credential, and tangential information that might have been
leaked by the <a>issuer</a>, such as the subject's home address. To mitigate such
leakage, <a>issuers</a> might choose to use common identifiers to mask specific 
location information or other sensitive metadata. For example,, a shared issuer 
identifier at a state or nation level, instead of at the level of a county, city, town, or other smaller
municipality. Further, <a>holder</a> attestation mechanisms can be used by
<a>verifiers</a> to preserve privacy, by providing proofs that an <a>issuer</a>
exists in a set of trusted entities, without needing to disclose the exact
<a>issuer</a>.
```